src/vde_backend/sync_flukso.py

```diff
@@ -732,6 +732,9 @@ def process_homes(tmpo_session, config, timings, now, custom):
         if timings[hid]["start_ts"] is not None and timings[hid]["end_ts"] is not None:
             # set init seconds (for tmpo query), might set timings
             # earlier than planned (not a problem)
+            # Skip the sync in the case where we want to sync a specific home(s)
+            if homes is not None and hid not in homes:
```

Comment:
Negative conditions are often ambiguous. A better way would be to run the sync only `if homes is None or hid in homes`.